python/ray/autoscaler/aws/cloudwatch/prometheus_ray_up.sh

```diff
@@ -21,19 +21,19 @@ if [ $? -eq 0 ]; then
   # TODO remove the version number hardcoding from this script
   # download the latest prometheus
   ray exec "$AUTOSCALER_CONFIG_FILE" "sudo wget https://github.com/prometheus/prometheus/releases/download/v2.25.0/prometheus-2.25.0.linux-amd64.tar.gz -P /opt"
-  ray exec "$AUTOSCALER_CONFIG_FILE" "cd /opt && sudo tar xvfz prometheus-2.25.0.linux-amd64.tar.gz"
+  ray exec "$AUTOSCALER_CONFIG_FILE" "cd /opt && sudo tar xvfz prometheus-2.25.0.linux-amd64.tar.gz && sudo mv prometheus-2.25.0.linux-amd64 prometheus && sudo rm prometheus-2.25.0.linux-amd64.tar.gz"
```

Comment:
Out of this PR's scope: shall we install the latest stable (ray-tested) version of Prometheus

Reply:
Based on @Zyiqin-Miranda's recent test results, it looks like we don't need to install Prometheus at all to scrape Ray system metrics, so I'll remove this in a subsequent revision.